newIDE/app/src/QuickCustomization/QuickCustomizationDialog.js

```diff
@@ -5,20 +5,25 @@ import { renderQuickCustomization, useQuickCustomizationState } from '.';
 import { Trans } from '@lingui/macro';
 import { type ResourceManagementProps } from '../ResourcesList/ResourceSource';
 import FlatButton from '../UI/FlatButton';
-import { ColumnStackLayout, LineStackLayout } from '../UI/Layout';
-import Text from '../UI/Text';
-import { useResponsiveWindowSize } from '../UI/Responsive/ResponsiveWindowMeasurer';
-import Paper from '../UI/Paper';
+import { ColumnStackLayout } from '../UI/Layout';
 import { type Exporter } from '../ExportAndShare/ShareDialog';
 import { useGameAndBuildsManager } from '../Utils/UseGameAndBuildsManager';
 import { sendQuickCustomizationProgress } from '../Utils/Analytics/EventSender';
 import ScrollView from '../UI/ScrollView';
+import PreviewLine from './PreviewLine';
+import UnsavedChangesContext from '../MainFrame/UnsavedChangesContext';
+
+const styles = {
+  scrollView: {
+    display: 'flex',
```

Comment:
I'm surprised we need this.

Reply:
me too, I think it wasn't looking great when only a few props. (as we're now hiding more props) I'll double check it's needed, but I think so.

Reply:
I've removed it but can't seem to see any issue. Weird 🤔